apps/jetbrains/jetbrains.py

```diff
@@ -7,7 +7,7 @@
 import requests
 from talon import Context, Module, actions, clip, ui
 
-# Courtesy of https://github.com/anonfunc/talon-user/blob/master/apps/jetbrains.py
+# Courtesy of https://github.com/anonfunc/talon-user/blob/main/apps/jetbrains.py
```

Comment:
This seems like a mistake; this isn't a knausj link, and the updated version 404s.

```suggestion
# Courtesy of https://github.com/anonfunc/talon-user/blob/master/apps/jetbrains.py
```